src/coreclr/src/libraries-native/CMakeLists.txt

```diff
@@ -1,11 +1,22 @@
 set(CMAKE_INCLUDE_CURRENT_DIR ON)
```

Comment:
I am wondering whether we need this directory at all. It kind of made sense when `entrypoints.c` lived in this directory. Now that we have entrypoints.c in each library, it does not make a lot of sense anymore.

Reply:
"libraries-native" folder is gone now. There is no need for it.